docs/source/features/ray.rst

```diff
@@ -0,0 +1,377 @@
+===========================
+Ray Job Dispatch and Tuning
+===========================
+
+.. currentmodule:: omni.isaac.lab
+
+Isaac Lab supports `Ray <https://docs.ray.io/en/latest/index.html>`_ for streamlining dispatching multiple training jobs (in parallel and in series),
+and hyperparameter tuning, both on local and remote configurations.
+
+This `independent community contributed walkthrough video <https://youtu.be/z7MDgSga2Ho?feature=shared>`_
+demonstrates some of the core functionality of the Ray integration covered in this overview. Although there may be some
+differences in the codebase (such as file names being shortened) since the creation of the video,
+the general workflow is the same.
+
+.. attention::
+
+  This functionality is experimental.
```

Comment:
let's also mention that it's currently linux only

Reply:
Added!